fe/fe-core/src/main/java/com/starrocks/memory/ProcProfileCollector.java

```diff
@@ -79,6 +79,7 @@ private void collectMemProfile() {
         String fileName = MEM_FILE_NAME_PREFIX + currentTimeString() + ".html";
         collectProfile(StarRocksFE.STARROCKS_HOME_DIR + "/bin/async-profiler/bin/asprof",
                 "-e", "alloc",
+                "--alloc", "2m",
```

Comment:
what does "2M bytes" mean? could you elaborate a little bit?

Reply:
Collect a sample after 2m bytes is allocated.